docs/flexibility/trigger-updates.md

```diff
@@ -0,0 +1,58 @@
+# Trigger updates
+
+Clients can issue simultaneous queries for a table. The [`table_open_cache`](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_table_open_cache) system variable returns the number of open tables for all threads. Raising this value increases the number of file descriptors. The [`table_open_cache_instances`](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_table_open_cache_instances) shows the number of open tables cache instances.
+
```

Comment:
Also I think it is a good idea to explain how triggers are related to open tables and why there is a problem with memory consumption.  Perhaps we can say something like:

When table with triggers defined on it is opened to be added to Table Cache, trigger definitions are parsed and the open table instance gets its own instances of trigger bodies. Thanks to this when connection that executes DML statement needs to run a trigger it gets its own instance of trigger body for execution (which is associated with the specific open table instance used by this connection). As result of such approach to trigger body caching open table instances in Table Cache for tables with triggers can consume surprising amounts of memory.

What do you think?

Reply:
Updated.